homeassistant/components/sensor/steam_online.py

```diff
@@ -21,12 +21,15 @@
 
 ICON = 'mdi:steam'
 
+STATE_OFFLINE = 'Offline'
```

Comment:
I think all these states should be lowercase snakecase. What do you think @armills?

Reply:
Are talking about the state values or the constant names?

Reply:
`'Offline'` should be `'offline'`.

Reply:
That I can change as well, get everything sorted at the same time.

Reply:
We probably should. We haven't been perfect about this in the past for custom sensor states since its also used for display, but we at least have a roadmap now for localizing these strings.

Then to clarify, the snakecase will mean `'Looking to trade'` will become `'looking_to_trade'`